ansible/playbooks/do_setup.yml

```diff
@@ -0,0 +1,242 @@
+---
+# this playbook will set up a kubernetes cluster, a mongodb database, create a spaces instance to run browsertrix cloud
+# It is assumed that you have a digital ocean API in your environment and have ACCESS keys for your Spaces
+# the playbook also uses doctl
+# set your env variables to have `DO_API_TOKEN` `DO_AWS_ACCESS_KEY` and `DO_AWS_SECRET_KEY`
+- name: deploy browsertrix on  digital ocrean
+  hosts: localhost
+  connection: local
+  gather_facts: false
+    #  vars_files:
+    # - "../group_vars/do/vault.yml"
+  vars:
+    - droplet_size: "s-1vcpu-2gb"
+    - droplet_region: "sfo3"
+    - do_project: "{{ my_project | default('btrixcloud-test')}}"
+    - space_name: "{{ my_space_name | default('btrixcloud-test')}}"
+    - btrix_db: "{{ my_db | default('btrixcloud-test')}}"
+    - k8s_cluster: "{{ my_k8s | default('btrixcloud-test')}}"
+    - domain_name: "{{ my_domain | default('example.edu')}}"
```

Comment:
For setting the `A` record, we probably want to split this to a `domain` and `subdomain` field, with `subdomain` defaulting to '@'?

For  setting the full domain in the app, will need to then combine subdomain + domain